README.md

```diff
@@ -1,70 +1,137 @@
+In a typical OpenShift cluster, there will be a cluster-scoped `ClusterVersion`
+resource called `version`:
 
-2. Run the following command to create release-image at `docker.io/abhinavdahiya/origin-release:latest`:
-
-```sh
-oc adm release new -n origin \
-    --from-image-stream=origin-v4.0 \
-    --to-image-base=docker.io/abhinavdahiya/origin-cluster-version-operator:latest \
-    --to-image docker.io/abhinavdahiya/origin-release:latest
+```console
+$ oc get clusterversion version
+NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+version   4.11.3    True        False         85d     Cluster version is 4.11.3
 ```
 
-## Installing CVO and operators in cluster.
+Note that as a user or a cluster administrator, you usually do not interact with
+the `ClusterVersion` resource directly but via either the [`oc adm upgrade`](https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/administrator-cli-commands.html#oc-adm-upgrade) CLI
+or the [web console](https://docs.openshift.com/container-platform/latest/updating/updating-cluster-within-minor.html#update-upgrading-web_updating-cluster-within-minor).
 
-1. Use CVO `render` to render all the manifests from release-payload to a directory. Refer the [Using CVO to render the release-image locally](#using-cvo-to-render-the-release-image-locally) section for more information.
+## Understanding Upgrades
 
-2. Create the operators from the manifests by using `oc create -f <directory when CVO rendered manifests>`.
+> :bulb: This section is only a high-level overview. See the [Update Process](https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/user/update-workflow.md) and [Reconciliation](https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/user/reconciliation.md) documents in the [dev-guide](https://github.com/openshift/enhancements/blob/master/dev-guide) for more details.
 
-## Using CVO to render the release-image locally
+The Cluster Version Operator continuously fetches information about upgrade
+paths for the configured channel from the OpenShift Update Service (OSUS). It
+stores the recommended update options in the `status.availableUpdates` field of
+its `ClusterVersion` resource.
 
-1. Run the following command to get render the release-payload contents to `/tmp/cvo`
+The intent to upgrade the cluster to another version is expressed by storing the
+desired version in the `spec.desiredUpdate` field, either manually by the
+administrator through the CLI or web console, or CVO does it automatically when
+automated upgrades are enabled.
 
-```sh
-podman run --rm -ti \
-    -v /tmp/cvo:/tmp/cvo:z \
-    <release image> \
-        render \
-        --output-dir=/tmp/cvo \
-        --release-image="<release image>"
-```
+When `spec.desiredUpdate` does not match the current cluster version, CVO will
+start updating the cluster. It downloads the release payload image, validates
```

Comment:
nit: I'm fine with this simplification, but when `spec.desiredUpdate` does not match the current cluster version, the CVO will consider updating the cluster.  We might refuse to update (e.g. if we cannot find a signature for the release image), although if  `force: true`, that waives all of our target-acceptance guards.

Reply:
I think it's fine to focus on the happy path. I mention the validation in the next sentence which IMO implies that it can fail.